calour/io.py

```diff
@@ -393,36 +414,99 @@ def read_amplicon(data_file, sample_metadata_file=None,
     sample_metadata_file : str or None (optional)
         None (default) to not load metadata per sample
-        str to specify name of sample mapping file (tsv)
+        str to specify name of sample mapping file (tsv).
+        Note: sample names in the bucket table and sample_metadata file must match. In case bucket table sample names contains additional
+        information, you can split them at the separator character (usually '_'), keeping only the first part, using the cut_sample_id_sep='_' parameter
+        (see below)
     gnps_file : str or None (optional)
-        name of the gnps clusterinfosummarygroup_attributes_withIDs_arbitraryattributes/XXX.tsv file
-        for use with the 'gnps' database in plot
+        name of the gnps clusterinfosummarygroup_attributes_withIDs_arbitraryattributes/XXX.tsv file, for use with the 'gnps' database.
+        This enables identification of the metabolites with known MS2 (for the interactive heatmap and sorting/filtering etc), as well as linking
+        to the gnps page for each metabolite (from the interactive heatmap - by double clicking on the metabolite database information).
+        Note: requires gnps-calour database interface module (see Calour installation instructions for details).
     feature_metadata_file : str or None (optional)
         Name of table containing additional metadata about each feature
         None (default) to not load
+    file_type: str, optional
+        the data file format. sets default values for the additional parameters in order to properly load the table (which can be overwritten). options include:
+        'mzmine2': load the mzmine2 output csv file.
+            MZ and RT are obtained from this file.
+            GNPS linking is direct via the unique id column.
+            table is csv, columns are samples.
+        'biom': load a biom table for the features
+            MZ and RT are obtained via the featureID (first column), which is assumed to be MZ_RT.
+            GNPS linking is indirect via the mz and rt threshold windows.
+            table is a tsv/json/hdf5 biom table, columns are samples.
+        'openms': load an openms output table
+            MZ and RT are obtained via the featureID (first column), which is assumed to be MZ_RT.
+            GNPS linking is indirect via the mz and rt threshold windows.
+            table is a csv table, columns are samples.
+        'gnps-ms2': load a gnps exported biom table
+            MZ and RT are obtained via the gnps_file if available, otherwise are NA
+            GNPS linking is direct via the first column (featureID).
+            table is a tsv/json/hdf5 biom table, columns are samples.
+    rows_are_samples: bool or None, optional
+        None (default) to use the file_type defaults.
+        False indicates columns are samples.
+        True indicates rows are samples.
+    table_sep: str or None, optional
```

Comment:
use `data` / `table` consistently

Reply:
fixed to data_table_sep
also noticed i didn't use it! so fixed (now passed to read() and from there to _read_csv()